frontend/src/metabase/components/CreateDashboardModal.jsx

```diff
@@ -127,7 +142,10 @@ export default class CreateDashboardModal extends Component {
                     errors={this.state.errors}
                   >
                     <Select
-                      value={this.state.collection_id}
+                      value={
+                        this.state.collection_id ||
+                        this.props.params.collectionId
```

Comment:
@tlrobinson Is there a good example somewhere of how to populate an initial value on selects? I'm running into issues with both the dashboard modal and the pulse collection selection when I pass the context of the current collection in.

Reply:
Generally we'd set the default state from the props in the constructor, i.e.

```javascript
constructor(props) {
  super(props);
  this.state = {
    collectionId: props.params.collectionId
  }
}
```

Of course that requires `collectionId` be a param on the current route (not just the "referring" route). Is that the case?